lib/collection/src/config.rs

```diff
@@ -146,6 +146,16 @@ impl CollectionConfig {
             validation::warn_validation_errors("Collection configuration file", errs);
         }
     }
+
+    pub fn get_quantization_named_configs(&self) -> BTreeMap<String, Option<QuantizationConfig>> {
```

Comment:
On the `Single` variant, we can still set a 'named' quantization configuration that is different than the global one.